src/main/java/seedu/address/logic/commands/AddCourseCommand.java

```diff
@@ -0,0 +1,70 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_COURSE;
+
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.course.Course;
+
+/**
+ * Adds a course to the course list.
+ */
+public class AddCourseCommand extends Command {
+
+    public static final String COMMAND_WORD = "add";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a course to the course list. "
+            + "Parameters: "
+            + PREFIX_COURSE + "COURSE ";
+
+
+    public static final String MESSAGE_SUCCESS = "New Course added: %1$s";
+    public static final String MESSAGE_DUPLICATE_Course = "This Course already exists in the Course list";
```

Comment:
Would it be better to use course instead of Course for lines 24-25?

Reply:
Thank you for the input, I agree and will edit this for the next PR